jxl_cli/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 [dependencies]
 jxl = { path = "../jxl" }
+clap = { version = "4.5.18", features = ["derive"] }
```

Comment:
I don't think you're actually using the derive feature...

Reply:
you are right: removed it